test/javascripts/unit/codebytes.test.js

```diff
@@ -92,8 +93,6 @@ describe('findCodeByte', () => {
 
   it('ignores codebytes with non-language attributes', () => {
     const testString = [
-      '[codebyte ]',
```

Comment:
I made the regex in findCodeByte a little more flexible to match the Discourse bbcode parser better. Extra whitespace is allowed as long as it's not between the opening `[` and the tag name.